client/components/AutocompleteInput/hooks/useAutocompleteInput.ts

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import debounce from 'debounce'
+
+import { parsePackageString } from '../../../../utils/common.utils'
+import API from '../../../api'
```

Comment:
Perhaps it could be a good idea to enable absolute imports at some point 😁

Reply:
I'd argue if absolute imports are a good idea, you can't enforce them in different IDEs. Unless there is a way to do that? I prefer monorepo approach with named package aliases with a single possible entry point.